src/runtime/storage-proxy.ts

```diff
@@ -553,6 +565,64 @@ export class BigCollectionProxy extends StorageProxy implements BigCollectionSto
   }
 }
 
+export class DummyProxy extends StorageProxy implements CollectionStore, BigCollectionStore, SingletonStore {
+  _getModelForSync(): {id: string;} | ModelValue[] {
+    return null;
+  }
+  _synchronizeModel(version: number, model: SerializedModelEntry[]): boolean {
+    return true;
+  }
+  _processUpdate(update: {version: number;}, apply?: boolean): {} {
+    return null;
+  }
+  reportExceptionInHost(exception: PropagatedException): void {}
+
+  deregister() {}
+
+  register() {}
+
+  _onSynchronize({version, model}: {version: number, model: SerializedModelEntry[]}): void {}
+
+  _onUpdate(update: {version: number}): void {}
+
+  _notify(kind: string, details, predicate = (ignored: HandleOptions) => true) {}
+
+  _processUpdates(): void {}
+
+  protected generateBarrier(): string {
+    return null;
+  }
+  async get(id?: string) {
+    return Promise.resolve();
+  }
+  // tslint:disable-next-line: no-any
+  async store(value: any, keys: string[], particleId?: string): Promise<void> {
+    return Promise.resolve();
```

Comment:
Should this return a resolved promise or a never resolved promise? I would say the latter, so that any async code waiting on this operation never executes. WDYT?

Reply:
Changed all of them into returning promises that never resolve. Just wondering, is it okay for those codes to just never executes into completion?